notifications_utils/clients/redis/bounce_rate.py

```diff
@@ -38,8 +44,24 @@ def set_notifications_seeded(self, service_id: str, seeded_data: dict) -> None:
     def set_hard_bounce_seeded(self, service_id: str, seeded_data: dict) -> None:
         self._redis_client.add_data_to_sorted_set(hard_bounce_key(service_id), seeded_data)
 
-    def get_bounce_rate(self, service_id: str, bounce_window=_twenty_four_hour_window_ms()) -> float:
+    def set_seeding_complete(self, service_id: str) -> None:
```

Comment:
do we need a seeding start key?

Reply:
do you mean a key that we would set once the seeding has started, so that another process doesn't try to do it if it is already in progress?

Reply:
correct. This won't return False? 
https://github.com/cds-snc/notification-api/pull/1834/files#diff-cca5917c620894fe770faa88835e9e186285d5b91a55fca992eb94d5dc87e635R823

Reply:
Yes that's a good point. If we have a "seeding_started" key then we wouldn't need a "seeding_complete" one since we can just check if any process has started seeding the data yet. I'll make that change.